gitbootcamp.rst

```diff
@@ -205,51 +201,48 @@ Commit the files:
 
 .. code-block:: bash
 
-   git commit -m '<message>'
-
+   git commit -m "<message>"
 
 .. _push-changes:
 
 Pushing Changes
 ---------------
 
-Once your changes are ready for a review or a pull request, you'll need to push
+Once your changes are ready for a review or a pull request, you will need to push
 them to the remote repository.
 
 ::
 
    git checkout <branch-name>
    git push origin <branch-name>
 
-
 Creating a Pull Request
 -----------------------
 
 1. Go to https://github.com/python/cpython.
 
-2. Press ``New pull request`` button.
+2. Press the ``New pull request`` button.
 
-3. Click ``compare across forks`` link.
+3. Click the ``compare across forks`` link.
 
-4. Select the base fork: ``python/cpython`` and base branch: ``master``.
+4. Select the base repository: ``python/cpython`` and base branch: ``master``.
 
-5. Select the head fork: ``<username>/cpython`` and base branch: the branch
+5. Select the head repository: ``<username>/cpython`` and head branch: the branch
```

Comment:
For step 5, this usage of "fork" is appropriate. I don't think this change is required.

Reply:
I disagree as GitHub uses exactly "base repository" and "head repository".